arch/arm/core/aarch32/cache.c

```diff
@@ -1,108 +1,100 @@
-/*
- * Copyright (c) 2013-2014 Wind River Systems, Inc.
- * Copyright (c) 2020-2022 Qualcomm Innovation Center, Inc.
- *
- * SPDX-License-Identifier: Apache-2.0
- */
+/* cache.c - cache support for AARCH32 CPUs */
+
```

Comment:
Before I dig into this in any depth, you shouldn't be removing the previous copyright here!

Reply:
Added SPDX-License-Identifier: Apache-2.0

I think as this is a delete+new rather than a modification, the previous copyright for Wind River and Qualcomm is void, isn't it? Tell me if I should keep those even though the implementation is not a modification.

Reply:
Please do not make unnecessary changes.

Reply:
If this is a complete rewrite, it still requires a copyright statement.

This also seems to differ significantly from what I see in `main`: https://github.com/zephyrproject-rtos/zephyr/blob/main/arch/arm/core/aarch32/cache.c (`arch_dcache_enable` etc.). I'll let @carlocaione follow up on the content, though, as the main author behind those changes.